l10n/en/firefox/browsers/mobile/android.ftl

```diff
@@ -50,3 +51,11 @@ mobile-android-pop-videos-out = Pop videos out of their websites or players and
 
 mobile-android-about-mozilla = About { -brand-name-mozilla }
 mobile-android-mozilla-exists-to = { -brand-name-mozilla } exists to build the Internet as a public resource accessible to all because we believe open and free is better than closed and controlled. We build products like { -brand-name-firefox } and { -brand-name-pocket } to promote choice, transparency and control.
+
+# 'Shortcut' is US slang, means a quicker way of doing or acheiving something. Being used here as a verb to mean "Take a shortcut..." Alternative is 'Get there faster with Firefox for Android'
+mobile-android-short-cut-the-internet = Shortcut the internet with { -brand-name-firefox } for { -brand-name-android }
+mobile-android-get-there-faster = Get there faster with { -brand-name-firefox } for { -brand-name-android }
```

Comment:
Where is this string used? There's no reference in the code

Reply:
Oops, this was a mistake. The alternate isn't actually used, it was just offered in the comment on the previous string. We shouldn't have included it in the file.